ui-v2/app/styles/components/filter-bar.scss

```diff
@@ -27,19 +20,3 @@ html.template-acl.template-list .filter-bar [role='radiogroup'] {
   color: $blue-500;
   background-color: $gray-100;
 }
-
-%filter-bar .value-passing span::before {
-  @extend %with-check-circle-fill-icon, %as-pseudo;
-}
-%filter-bar .value-warning span::before {
-  @extend %with-alert-triangle-icon, %as-pseudo;
-}
-%filter-bar .value-critical span::before {
-  @extend %with-cancel-square-fill-icon, %as-pseudo;
-}
-%filter-bar .value-allow span::before {
-  @extend %with-arrow-right-icon, %as-pseudo;
-}
-%filter-bar .value-deny span::before {
-  @extend %with-deny-icon, %as-pseudo;
-}
```

Comment:
We no longer need all these icons now we have the new filter bar dropdown filters.